MonoGame.Framework/Utilities/OS.cs

```diff
@@ -0,0 +1,46 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace MonoGame.Utilities
+{
+    /// <summary>
+    /// Defines the operating system.
+    /// </summary>
+    public enum OS
```

Comment:
That seems pretty game-specific and I'm not sure it warrants adding yet another way to check "platform" when there's already the platform enumeration provided by MonoGame, and the .NET framework provides several ways to detect it at runtime.